### PR TITLE
Pronoun Roles

### LINKED ISSUE #21: [FEAT] Pronoun Roles
**Is your feature request related to a problem? Please describe.**
NO
**Describe the solution you'd like**
make pronoun roles!

**Describe alternatives you've considered**
 NONE

**Additional context**

Who would you like to work on this issue?

NINO

### PR DESCRIPTION
**Pull request by egelja**
Closes #21

### CODE REVIEW

File: roycemorebot/exts/pronouns.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    """User-assigned roles based on their grade."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.guild_only()
+    @commands.command(name="he-him", aliases=("he", "him", "hehim"))
+    async def he_him(self, ctx: commands.Context) -> None:
+        """Self-give the `He/Him` Role."""
+        # User is switching roles, remove all current roles
```
Comment (AlemSnyder) [inline roycemorebot/exts/pronouns.py]: no only toggle

File: roycemorebot/exts/pronouns.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        if await has_any_role_check(ctx, *PRONOUN_ROLES):
+            for role in PRONOUN_ROLES:
+                await ctx.author.remove_roles(
```
Comment (AlemSnyder) [inline roycemorebot/exts/pronouns.py]: why not `try remove(he-him role)`

**Review by AlemSnyder — CHANGES_REQUESTED** [review]
people should be able to have multiple pronoun roles

File: roycemorebot/exts/pronouns.py (hunk 2 above)
Comment (egelja) [reply roycemorebot/exts/pronouns.py]: You don't need the `try`.

**Review by AlemSnyder — APPROVED** [review]

**Pull request merged**